### DISCUSSION TITLE
MahApps UI framework test

### DISCUSSION THREAD

**Discussion by VShawn** [1Remote/1Remote#47] [_Development]
# I try MahApps:https://mahapps.com/

everything looks good:

## controls
![image](https://user-images.githubusercontent.com/10143738/103182443-37285480-48e6-11eb-873d-68d38b40e5ac.png)

## notifications
![image](https://user-images.githubusercontent.com/10143738/103182477-87071b80-48e6-11eb-9d8b-ae332c4bfc04.png)

## issues

It just getting too big, it groth to 36.6M whithout others dll while the current PRemoteM is 8.53M.

too many icons in it.

![image](https://user-images.githubusercontent.com/10143738/103182531-ec5b0c80-48e6-11eb-8e78-58030304c561.png)

**Comment by majkinetor** [comment]
So what is used now for UI?

And why not MAUI?

**Reply by VShawn** [reply]
> So what is used now for UI?

1RM used my personal custom UI lib now. 

> And why not MAUI?

It's now a good idea to use MAUI when 2020. I don't know how MAUI is now, but about 6 months ago that version still had a lot of bugs.

**Reply by majkinetor** [reply]
Yeah? Zero experience with it, I just know that with .NET 7 it is in production mode. Its not terribly important now since 1RM is tied to Windows.

**Reply by VShawn** [reply]
Zero experience with it too, there are many bug ticket in MAUI repository and bugs become more and more...

6 months age there 800+ bug tickets now 1500+

https://github.com/dotnet/maui/issues?q=is%3Aissue+is%3Aopen+label%3At%2Fbug

**Reply by majkinetor** [reply]
OK, so I guess its not ready yet then.
